lib/src/install.rs

```diff
@@ -927,6 +890,9 @@ async fn prepare_install(
 
     let skopeo_supports_containers_storage = skopeo_supports_containers_storage()
         .context("Failed to run skopeo (it currently must be installed in the host root)")?;
+    if !skopeo_supports_containers_storage {
+        anyhow::bail!("skopeo is too old");
```

Comment:
I think  the `skopeo_supports_containers_storage` should error out with a message indicating the current / expected version of Skeop. "too old" will put the user on a journey figuring out the requirement.